docs/03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md

```diff
@@ -0,0 +1,288 @@
+6. Provide the `Event type` under `Filter 2` as `sap.kyma.custom.myapp.order.changed.v1`. Leave the `Event source` as empty.
+
+   > **NOTE:** You can add more filters to your subscription if you want to subscribe to more event types.
+
+7. Click **Create**.
+8. Wait a few seconds for the Subscription to have status `READY`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+```bash
+cat <<EOF | kubectl apply -f -
+    apiVersion: eventing.kyma-project.io/v1alpha1
+    kind: Subscription
+    metadata:
+      name: lastorder-sub
+      namespace: default
+    spec:
+      sink: 'http://lastorder.default.svc.cluster.local'
+      filter:
+        filters:
+          - eventSource:
+              property: source
+              type: exact
+              value: ''
+            eventType:
+              property: type
+              type: exact
+              value: sap.kyma.custom.myapp.order.received.v1
+          - eventSource:
+              property: source
+              type: exact
+              value: ''
+            eventType:
+              property: type
+              type: exact
+              value: sap.kyma.custom.myapp.order.changed.v1
+EOF
+```
+
+To check that the Subscription was created and is ready, run:
+```bash
+kubectl get subscriptions lastorder-sub -o=jsonpath="{.status.ready}"
+```
+
+The operation was successful if the returned status says `true`.
+
+  </details>
+</div>
+
+## Trigger the workload with an event
+
+We created the `lastorder` Function, and subscribed to the `order.received.v1` and `order.changed.v1` events by creating a Subscription CR. Now it's time to publish the events and trigger the Function.
+In this example, we'll port-forward the Kyma Eventing Service to localhost.
+
+1. Port-forward the Kyma Eventing Service to localhost. We will use port `3000`. Run:
+   ```bash
+   kubectl -n kyma-system port-forward service/eventing-event-publisher-proxy 3000:80
+   ```
+2. Publish an event of type `order.received.v1` to trigger your function. In another Terminal window run:
+
+    <div tabs name="Publish an event" group="trigger-workload">
+      <details open>
+      <summary label="CloudEvents Conformance Tool">
+      CloudEvents Conformance Tool
+      </summary>
+    
+       ```bash
+       cloudevents send http://localhost:3000/publish \
+          --type sap.kyma.custom.myapp.order.received.v1 \
+          --id cc99dcdd-6f6d-43d6-afef-d024eb276584 \
+          --source myapp \
+          --datacontenttype application/json \
+          --data "{\"orderCode\":\"3211213\", \"orderStatus\":\"received\"}" \
+          --yaml
+       ```
+    
+      </details>
+      <details>
+      <summary label="curl">
+      curl
+      </summary>
+    
+       ```bash
+       curl -v -X POST \
+            -H "ce-specversion: 1.0" \
+            -H "ce-type: sap.kyma.custom.myapp.order.received.v1" \
+            -H "ce-source: /default/io.kyma-project/custom" \
```

Comment:
why is the source here a different source than in the conformance tool command?

Reply:
updated